package.json

```diff
@@ -112,5 +112,8 @@
     "react-dropzone": "14.2.3",
     "react-router-dom": "^6.0.0",
     "vanilla-framework": "^4.6.0"
+  },
+  "dependencies": {
+    "@tanstack/react-table": "^8.15.3"
```

Comment:
Let's add this as a peerDependency. We expect react-table to be provided and managed by each project consuming the library. This helps avoid potential version conflicts.